src/pkgcheck/checks/network.py

```diff
@@ -274,6 +274,7 @@ def __init__(self, *args, **kwargs):
             'gentoo': 'https://gitweb.gentoo.org/{project}.git/',
             'github': 'https://github.com/{project}',
             'gitlab': 'https://gitlab.com/{project}',
+            'hackport': 'https://hackage.haskell.org/package/{project}',
```

Comment:
This really should be `'hackage': ...`

hackport is just a project of gentoo-haskell, a portmanteau of "hackage" and "portage".

Reply:
I just made a typo.

Reply:
ping?

Reply:
Will fix shortly

Reply:
done